server/s3_object_metadata.cc

```diff
@@ -872,6 +872,11 @@ void S3ObjectMetadata::set_tags(
   object_tags = tags_as_map;
 }
 
+void S3ObjectMetadata::set_bucket_replication_configuration(
```

Comment:
check this function and correct if necessary

Reply:
Removed the function.